pkg/sql/conn_executor_savepoints.go

```diff
@@ -147,6 +147,12 @@ func (ex *connExecutor) execRelease(
 		res.ResetStmtType((*tree.CommitTransaction)(nil))
 		err := ex.commitSQLTransactionInternal(ctx, s)
 		if err == nil {
```

Comment:
Is there a less nested way to do this? I.e. can we test err != nil instead?

Easy to miss that one of those returns is for the anonymous function instead of the parent, and that we are shadowing `err`.